index.html

```diff
@@ -64,6 +64,7 @@ <h3>
           </h3>
           <p>
             The most common values will be objects or arrays, but any JSON value is permitted.
+            As reminded in issue <a href="https://github.com/wardi/jsonlines/issues/100#issuecomment-2897805072">#100</a>, empty lines are not valid JSON.
```

Comment:
I'd rather not link to a discussion on the repo, how about
```suggestion
            e.g. <code>null</code> is a valid value but a blank line is not.
```